[EV-4240] Remove references to values.yaml replacement (#1028)
tigera/docs: calico-enterprise/multicluster/helm/create-a-management-cluster-helm.mdx

  export MANAGED_CLUSTER_CERTIFICATE=<managed-cluster-certificate>
  ```

> 1. Verify the management cluster context has been correctly updated in the `values.yaml`. For a NodePort:

Reviewer: ```suggestion
1. Verify that the management cluster context is correctly updated in the `values.yaml`. For a NodePort:
```
